tutorials/single_node_tutorial/single_gpu_tutorial.ipynb

```diff
@@ -0,0 +1,3423 @@
+   "source": [
+    "res = download_wikipedia(download_output_directory,\n",
+    "                   language=language, \n",
+    "                   dump_date=dump_date,\n",
+    "                   url_limit=url_limit).df.compute()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2aae29dd",
+   "metadata": {},
+   "source": [
+    "Verify result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "169fadb9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "downloads  thwiki-20240201-pages-articles-multistream.xml.bz2.jsonl\n",
+      "162164 /nluo_data/NeMo-Curator/tutorials/single_node_tutorial/workspace/wiki_downloads/data/thwiki-20240201-pages-articles-multistream.xml.bz2.jsonl\n"
+     ]
+    }
+   ],
+   "source": [
+    "! ls {download_output_directory}\n",
+    "! wc -l  {download_output_directory}/thwiki-20240201-pages-articles-multistream.xml.bz2.jsonl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "f2bcb168",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"text\":\"–\\n\\nป้ายบอกทาง \\n ศาลาประชาคม – กระดานข่าว โครงการ ทรัพยากรและกิจกรรมซึ่งครอบคลุมวิกิพีเดียอย่างกว้างขวาง\\n แผนกช่วยเหลือ – ถามข้อสงสัยเกี่ยวกับการใช้งานวิกิพีเดีย\\n ปุจฉา-วิสัชนา – ถามข้อสงสัยทั่วไปที่คุณอยากรู้\\n ข่าวไซต์ – ประกาศ อัพเดต บทความและข้อมูลข่าวเกี่ยวกับวิกิพีเดียและมูลนิธิวิกิมีเดีย\\n สภากาแฟ – สำหรับอภิปรายเกี่ยวกับวิกิพีเดีย รวมถึงรายงานปัญหาเทคนิคและเสนอนโยบาย\\n Local Embassy – For Wikipedia-related discussion in languages other than Thai.\\n สร้างบทความใหม่ – บทช่วยสอนสำหรับเตรียมพร้อมสร้างบทความแรกของคุณ\\n\\nภาษาอื่น \\n\\n \",\"title\":\"หน้าหลัก\",\"id\":\"1\",\"url\":\"https:\\/\\/th.wikipedia.org\\/wiki\\/%E0%B8%AB%E0%B8%99%E0%B9%89%E0%B8%B2%E0%B8%AB%E0%B8%A5%E0%B8%B1%E0%B8%81\",\"language\":\"th\",\"source_id\":\"thwiki-20240201-thwiki-20240201-pages-articles-multistream.xml.bz2\",\"filename\":\"thwiki-20240201-pages-articles-multistream.xml.bz2.jsonl\"}\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "check_jsonl_file(download_output_directory)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44fa2d13",
+   "metadata": {},
+   "source": [
+    "**[Optional]**Close the Dask cluster.You might encounter error such as `Caught signal 11`.It's OK, just rerun the cell again."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "590c489c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# client.cluster.close()\n",
+    "# client.shutdown()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5ba566fc",
+   "metadata": {},
+   "source": [
+    "## 2.Language separation and unicode fixing\n",
+    "\n",
+    "**Note**: In order to be run on interactive python. Please comment `from.code import *` and the related imports in `./nemo_curator/filters/__init__.py`"
```

Comment:
Can you try this again with the latest version of curator? And please let us know what errors you get if you get any.

Reply:
This is  fixed. I will amend the notebook accordingly